packages/wonder-blocks-modal/docs.md

```diff
@@ -1,10 +1,29 @@
 Looking for docs for StandardModal, OneColumnModal, or TwoColumnModal click [here](https://deploy-preview-389--wonder-blocks.netlify.com/#modal)
 
-## Wrapping Modals
+## Accessibility
 
-Often you'll want to define a new modal component by wrapping an existing
-modal component.  These wrapped components will also work with ModalLauncher
-in the same way the default ones do.
+It should follow guidelines from [W3C](https://www.w3.org/TR/wai-aria-practices/#dialog_modal).
+
+### Keyboard Interaction
+When a dialog opens, focus moves to an element inside the dialog. See notes below regarding initial focus placement.
```

Comment:
It's probably worth mentioning the behavior of focus when closing the modal.

Reply:
Good call. What's the current behavior? Are we currently returning the focus to the element that opened the modal in the fist place?

Reply:
I thought we were, but I just tested and we aren't.  We should though.  Can you file a ticket for this?

Reply:
https://khanacademy.atlassian.net/browse/WB-564